src/settings/LostItemFeePolicy/components/ViewSections/LostItemFeeSection/LostItemFeeSection.test.js

```diff
@@ -0,0 +1,144 @@
+import React from 'react';
+import { noop } from 'lodash';
+import {
+  render,
+  screen,
+  within,
+} from '@testing-library/react';
+
+import '../../../../../../test/jest/__mock__';
+
+import LostItemFeeSection from './LostItemFeeSection';
+
+const testSectionKeyValues = (testName, testId, expectedValue) => {
+  it(`should render ${testName} with ${expectedValue} value`, () => {
+    const testIdValue = screen.getByTestId(testId);
+
+    expect(testIdValue).toBeVisible();
+    expect(within(testIdValue).getByText(expectedValue)).toBeVisible();
```

Comment:
Can we add test for test label here? Just for example: expect(getItemByTestId('sectionTestId').getByText(labelSelector)).toBeVisible();

Reply:
Done